bugsnag-android-core/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -131,20 +134,6 @@ private String calculateNotifierType() {
         }
     }
 
-    /**
-     * The version code of the running Android app, from android:versionCode
-     * in AndroidManifest.xml
-     */
-    @Nullable
-    @SuppressWarnings("deprecation")
-    private Integer calculateVersionCode() {
-        if (packageInfo != null) {
-            return packageInfo.versionCode;
-        } else {
-            return null;
-        }
-    }
```

Comment:
As I understand it, versionCode should behave exactly like appVersion: it can be included in the manifest under Bugsnag metadata, supplied programatically on `Configuration`, or otherwise it's read from the manifest under a common Android location. If so, the two fields should have the same logic in `Client` and `AppData`.

Reply:
Agreed to alter the implementation to follow the behaviour of appVersion for now. A ticket has been created to alter the implementation more generally as part of the immutable config work, to make Configuration the source of truth.